explainer.md

```diff
@@ -96,6 +96,50 @@ become [much more expensive](https://pq-crystals.org/dilithium/). The
 flexibility will also simplify the post-quantum transition itself, as different
 post-quantum CAs may be added at different times.
 
+### Minimizing Server Operator Burden
+
+In reasoning through PKI deployment strategies, one of our primary goals is to
+minimize server operator burden. Where possible, permit server operators to make
+simpler, high-level decisions (e.g. which ACME endpoint(s) cover the clients
+they need), with automation handling the rest.
+
+This is important because there are many HTTPS servers on the web, operated by
+different parties. While some strategies, such as fingerprinting or manually
+collecting an optimal set of intermediates, can be done by a small number of
+large hosting providers, they are impractical for a more diverse set of HTTPS
+servers. Avoiding this overhead helps both support the current server diversity
```

Comment:
Since we're not spending much time on this topic, I think we should also point out the technical limitations. Fingerprinting is relatively unreliable because it's an inferred signal and collecting intermediates requires constant curation to remain relevant. Maybe something like:
"they are impractical for a more diverse set of HTTPS servers and are limited in both the types of signals available and the reliability of those signals."